loopin.go

```diff
@@ -429,11 +429,11 @@ func resumeLoopInSwap(_ context.Context, cfg *swapConfig,
 	return swap, nil
 }
 
-// validateLoopInContract validates the contract parameters against our request.
-func validateLoopInContract(height int32, response *newLoopInResponse) error {
+// ValidateLoopInContract validates the contract parameters against our request.
+func ValidateLoopInContract(height int32, htlcExpiry int32) error {
 	// Verify that we are not forced to publish a htlc that locks up our
 	// funds for too long in case the server doesn't follow through.
-	if response.expiry-height > MaxLoopInAcceptDelta {
+	if htlcExpiry-height > MaxLoopInAcceptDelta {
```

Comment:
I propose to also check the lower bound. E.g. that `htlcExpiry > height` or even `htlcExpiry > height + something`.

Reply:
done